cypress/integration/Login/login.spec.js

```diff
@@ -0,0 +1,50 @@
+import LoginPage from "../../Pages/Login";
+
+describe("Login Page", () => {
+  beforeEach(() => {
+    LoginPage.open();
+  });
+
+  it("should pass healthchecks", () => {
+    LoginPage.emailInput.should("be.visible");
+    LoginPage.passwordInput.should("be.visible");
+    LoginPage.loginButton.should("be.visible");
+  });
+
+  it("should successfully login with valid credentials", () => {
+    const validCredentials = {
+      email: "hadarziv@sg.com",
+      password: "papa",
+    };
+    const { email, password } = validCredentials;
+
+    LoginPage.login(email, password).then(() => {
+      // TODO: Eventually we would refer to the bounce rules page object's getter here instead
+      expect("[data-test='bounce-rules-page']").should("be.visible");
```

Comment:
Keeping the main assertions in the spec file and chaining the page object's login function to be more readable